prefix_finder/frontend/templates/home.html

```diff
@@ -7,53 +7,53 @@
             </select>
-            {% if lookups.subnational %}
+            {% if lookups.substructure %}
               <div class="indented-option">
-                <div class="select-label">Subnational coverage</div>
-                <select name="subnational" class="selectpicker" data-live-search="true" data-width="280px" onchange="this.form.submit()">
-                <option value="">Any subnational location</option>
-                {% for code, title, disabled in lookups.subnational %}
-                  <option {% if query.subnational == code %} selected {% elif disabled %} disabled {% endif %} value="{{code}}">{{title}}</option>
+                <div class="select-label">In particular...</div>
+                <select name="substructure" class="selectpicker" data-live-search="true" data-width="280px" onchange="this.form.submit()">
+                <option value="">Any type</option>
+                {% for code, title, disabled in lookups.substructure %}
+                  <option {% if query.substructure == code %} selected {% elif disabled %} disabled {% endif %} value="{{code}}">{% if title|slice:"1" in "A,E,I,O,U"|slice:',' %}An{%else%}A{%endif%} {{title|lower}}</option>
                 {% endfor %}
                 </select>
               </div>
             {% endif %}
           </div>
+
           <div class="form-group col-md-4">
-            <div class="select-label">Organization type</div>
-            <select name="structure" class="selectpicker has-suboption" data-live-search="true" data-width="300px">
-              <option value="">Any type</option>
-              {% for code, title, disabled in lookups.structure %}
-                <option {% if query.structure == code %} selected {% elif disabled %} disabled {% endif %} value="{{code}}">{{title}}</option>
+            <div class="select-label">Registered in...</div>
+            <select name="coverage" class="selectpicker has-suboption" data-live-search="true" data-width="300px">
+              <option value="">Any country</option>
+              {% for code, title, disabled in lookups.coverage %}
+              <option {% if query.coverage == code %} selected {% elif disabled %} disabled {% endif %} value="{{code}}">{{title}}</option>
               {% endfor %}
             </select>
-            {% if lookups.substructure %}
+            {% if lookups.subnational %}
               <div class="indented-option">
-                <div class="select-label">Organisation subtype</div>
-                <select name="substructure" class="selectpicker" data-live-search="true" data-width="280px" onchange="this.form.submit()">
-                <option value="">Any subtype</option>
-                {% for code, title, disabled in lookups.substructure %}
-                  <option {% if query.substructure == code %} selected {% elif disabled %} disabled {% endif %} value="{{code}}">{{title}}</option>
+                <div class="select-label">In the state of...</div>
```

Comment:
I find this label a bit odd in some case, i.e. for UK you are prompted to choose a state